config/crd/bases/infrastructure.cluster.x-k8s.io_openstackclusters.yaml

```diff
@@ -1264,6 +1264,11 @@ spec:
                               type: array
                             description:
                               type: string
+                            disablePortSecurity:
+                              description: DisablePortSecurity disables the port security
```

Comment:
Where this description come from? Description should be auto generated from a comment in api/v1alpha4/types.go.

Reply:
You are right. I did not re-generate the yaml files after removing the comment. 
Will add the comment back.